wagtailio/core/blocks.py

```diff
@@ -196,24 +196,17 @@ def url(self):
 
 
 class CTALinkMixin(blocks.StructBlock):
-    def is_link_required(self):
```

Comment:
We actually have the required value passed to Streamfield, we can use that instead.